innovation.game.php

```diff
@@ -22829,7 +22886,28 @@ function stInterInteractionStep() {
                     self::executeDraw($player_id, $value_to_score, 'score');
                     self::executeDraw($player_id, $value_to_score, 'score');
                     break;     
-                
+                // id 413, Echoes age 8: Crossword
+                case "413N1A":
+                    $bonus_counts = self::getAuxiliaryArray();
+                    $remaining_values = count(array_filter($bonus_counts));
+                    // if more than 1 value remains, repeat the choice loop
+                    if($remaining_values > 1) {
+                        $step--;
```

Comment:
This part looks good.  You need to decrement the step to return to the current interaction.